pkg/manifest/context.go

```diff
@@ -99,7 +100,21 @@ func (c *Context) HasBucket(bucket string) bool {
 	}
 
 	return false
-} 
+}
+
+func (c *Context) AddDomain(bucket string) {
+	c.Domains = append(c.Domains, bucket)
+}
+
+func (c *Context) HasDomain(domain string) bool {
+	for _, d := range c.Domains {
+		if d == domain {
```

Comment:
can you add an exclusion here for `""`, there are some cases where an empty string for the hostname is used and we wouldn't want that to fail the uniqueness check.

Reply:
I've added this